client/src/components/Sidebar/Chat.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import { withStyles } from "@material-ui/core/styles";
+import {
+  List, ListItem, ListItemText,
+} from "@material-ui/core";
+import PropTypes from 'prop-types';
+
+import {
+  OwnMessage, OtherMessage, EventMessage, NotificationMessage,
+} from "./Messages";
+
+const mockChat = [
+  {type: "chat", author: "Bonnie", text: "I vote for dog"},
+  {type: "action", author: "Bonnie", text: "voted for 'Dog'"},
+  {type: "action", author: "Kayla", text: "voted for 'Dog'"},
+  {type: "chat", author: "Kayla", text: "Me to"},
+  {type: "chat", author: "Tommy", text: "Why dog?"},
+  {type: "chat", author: "Bonnie", text: "Dog is an animal!"},
+  {type: "action", author: "Tommy", text: "voted for 'Dog'"},
+  { type: "notification", text: "All team voted for 'Dog'", },
+];
+
+const styles = theme => ({
+  window: {
+    overflowY: "scroll",
+    height: "auto",
+    maxHeight: "45vh",
+    marginTop: theme.spacing(1),
+  },
+  nopadding: {
+    paddingTop: 0,
+    paddingBottom: theme.spacing(1),
+  },
+  floatRight: {
+    float: "right",
+  }
+});
+
+const Chat = ({ player, classes }) => {
+  const [chats, ] = useState(mockChat);
+  const isSelf = author => author === player ? "You" : author;
```

Comment:
are author and player objects with multiple properties? In that scenario it might be better to compare something like `author.id === player.id` instead of the whole object.

Reply:
author and player are both strings